libs/openFrameworks/graphics/ofPixels.cpp

```diff
@@ -1322,23 +1323,23 @@ bool ofPixels_<PixelType>::resizeTo(ofPixels_<PixelType>& dst, ofInterpolationMe
 
 			float srcColor = 0;
 			float interpCol;
-			int patchRow;
-			int patchIndex;
+			size_t patchRow;
+			size_t patchIndex;
 			float patch[16];
 
-			int srcRowBytes = srcWidth*bytesPerPixel;
-			int loIndex = (srcRowBytes)+1;
-			int hiIndex = (srcWidth*srcHeight*bytesPerPixel)-(srcRowBytes)-1;
+			size_t srcRowBytes = srcWidth*bytesPerPixel;
+			size_t loIndex = (srcRowBytes)+1;
+			size_t hiIndex = (srcWidth*srcHeight*bytesPerPixel)-(srcRowBytes)-1;
 
-			for (int dsty=0; dsty<dstHeight; dsty++){
-				for (int dstx=0; dstx<dstWidth; dstx++){
+			for (size_t dsty=0; dsty<dstHeight; dsty++){
+				for (size_t dstx=0; dstx<dstWidth; dstx++){
 
-					int   dstIndex0 = (dsty*dstWidth + dstx) * bytesPerPixel;
+					size_t   dstIndex0 = (dsty*dstWidth + dstx) * bytesPerPixel;
 					float srcxf = srcWidth  * (float)dstx/(float)dstWidth;
 					float srcyf = srcHeight * (float)dsty/(float)dstHeight;
-					int   srcx = (int) MIN(srcWidth-1,   srcxf);
-					int   srcy = (int) MIN(srcHeight-1,  srcyf);
-					int   srcIndex0 = (srcy*srcWidth + srcx) * bytesPerPixel;
+					size_t   srcx = static_cast<size_t>(MIN(srcWidth-1,   srcxf));
+					size_t   srcy = static_cast<size_t>(MIN(srcHeight-1,  srcyf));
```

Comment:
i see why now - the zero is read as "int" - if I cast the zero again it works fine
size_t minX = std::max(x, static_cast<size_t>(0));